ios/Exponent/Kernel/ReactAppManager/EXReactAppExceptionHandler.m

```diff
@@ -11,6 +11,8 @@
 #import <React/RCTBridge.h>
 #import <React/RCTRedBox.h>
 
+#import "Expo_Go-Swift.h"
```

Comment:
Is this necessary? There are no other changes in this file, so I guess it's just a leftover?